packages/mediacenter/kodi/patches/aarch64/kodi-0007-improve-sysinfo-for-AML-aarch64.patch

```diff
@@ -0,0 +1,54 @@
+From 4f24a459519c655c2dcd254c18a06f81cc226c24 Mon Sep 17 00:00:00 2001
+From: Jamie Coldhill <wrxtasy@amnet.net.au>
+Date: Wed, 15 Feb 2017 09:45:43 +0800
+Subject: [PATCH] kodi: improve sysinfo for AML aarch64
+
+---
+ xbmc/windows/GUIWindowSystemInfo.cpp | 11 +++--------
+ 1 file changed, 3 insertions(+), 8 deletions(-)
+
+diff --git a/xbmc/windows/GUIWindowSystemInfo.cpp b/xbmc/windows/GUIWindowSystemInfo.cpp
+index 93e9f74..1874bb4 100644
+--- a/xbmc/windows/GUIWindowSystemInfo.cpp
++++ b/xbmc/windows/GUIWindowSystemInfo.cpp
+@@ -109,7 +109,6 @@ void CGUIWindowSystemInfo::FrameMove()
+     SetControlLabel(i++, "%s %s", 13283, SYSTEM_OS_VERSION_INFO);
+     SetControlLabel(i++, "%s: %s", 12390, SYSTEM_UPTIME);
+     SetControlLabel(i++, "%s: %s", 12394, SYSTEM_TOTALUPTIME);
+-    SetControlLabel(i++, "%s: %s", 12395, SYSTEM_BATTERY_LEVEL);
+   }
+ 
+   else if (m_section == CONTROL_BT_STORAGE)
+@@ -132,8 +131,6 @@ void CGUIWindowSystemInfo::FrameMove()
+     SetControlLabel(i++, "%s: %s", 150, NETWORK_IP_ADDRESS);
+     SetControlLabel(i++, "%s: %s", 13159, NETWORK_SUBNET_MASK);
+     SetControlLabel(i++, "%s: %s", 13160, NETWORK_GATEWAY_ADDRESS);
+-    SetControlLabel(i++, "%s: %s", 13161, NETWORK_DNS1_ADDRESS);
```

Comment:
Is this for VPN's ?
As I've never seen DNS servers ever on shown aarch64 platforms.